i18n/locales/en.json

```diff
@@ -34,6 +34,7 @@
   "components.header.buttons.toggleDarkMode": "Toggle dark/light mode",
   "components.pagination.next": "Newer | ",
   "components.pagination.previous": "Older",
+  "components.common.banner.button.text": "Read More",
```

Comment:
**Important:** New translation keys should be added always at the end of the locale file.